StackExchange.Profiling.Wcf/Helpers/WcfOperationContextExtension.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace StackExchange.Profiling.Wcf.Helpers
+{
+    using System;
+    using System.Collections;
+    using System.Diagnostics.Contracts;
+    using System.ServiceModel;
+
+    /// <summary>
+    /// <c>Taken from http://blog.caraulean.com/2008/02/13/httpcontext-idiom-for-windows-communication-foundation/</c>
+    /// </summary>
+    internal class WcfOperationContextExtension : IExtension<OperationContext>
+    {
+        /// <summary>
+        /// The items.
+        /// </summary>
+        private readonly IDictionary _items;
```

Comment:
1. Why define _items as `IDicitionary` and initialize it to `HashTable`?
2. Why not initialize it in the definition line?